website/src/components/Chat/ChatListBase.tsx

```diff
@@ -98,7 +98,13 @@ export const ChatListBase = memo(function ChatListBase({
         }}
       >
         {chats.map((chat) => (
-          <ChatListItem key={chat.id} chat={chat} onUpdateTitle={handleUpdateTitle} onHide={handleHide} />
+          <ChatListItem
+            key={chat.id}
+            chat={chat}
+            onUpdateTitle={handleUpdateTitle}
+            onHide={removeItemFromList}
+            onDelete={removeItemFromList}
```

Comment:
We still need to redirect user to another page after deleting, maybe latest chat or /chat page

Reply:
Already implemented, maybe not in the best way, because you can delete a chat even though you haven't opened it, so we don't redirect always, we only redirect when we get a 404 on the chat.